cpp/src/groupby/hash/groupby.cu

```diff
@@ -96,11 +96,11 @@ bool constexpr is_hash_aggregation(aggregation::Kind t)
   // this is a temporary fix due to compiler bug and we can resort back to
   // constexpr once cuda 10.2 becomes RAPIDS's minimum compiler version
   // return array_contains(hash_aggregations, t);
-  return (t == aggregation::SUM) or (t == aggregation::MIN) or (t == aggregation::MAX) or
-         (t == aggregation::COUNT_VALID) or (t == aggregation::COUNT_ALL) or
-         (t == aggregation::ARGMIN) or (t == aggregation::ARGMAX) or
-         (t == aggregation::SUM_OF_SQUARES) or (t == aggregation::MEAN) or
-         (t == aggregation::STD) or (t == aggregation::VARIANCE);
+  return (t == aggregation::SUM) or (t == aggregation::PRODUCT) or (t == aggregation::MIN) or
```

Comment:
Oh, now we can use the `hash_aggregations` array now that we dropped 10.2!

Reply:
gpuCI still shows 10.2 builds.

Reply:
They are in the process of getting dropped for 0.20.